web/components/answers/answers-panel.tsx

```diff
@@ -52,7 +52,11 @@ export function AnswersPanel(props: {
 
   const chosenTotal = _.sum(Object.values(chosenAnswers))
 
-  const answerItems = getAnswers(contract, user)
+  const answerItems = getAnswers(
+    contract,
+    user,
+    winningAnswers.length > 0 ? winningAnswers.map((a) => a.id) : []
```

Comment:
Why not just filter the answers to exclude winning answers, instead of modifying getAnswers to take another param?

Reply:
because this is so easy!